crates/starknet-devnet-core/src/constants.rs

```diff
@@ -23,7 +23,7 @@ pub const CAIRO_1_ERC20_CONTRACT_PATH: &str =
     concat!(env!("CARGO_MANIFEST_DIR"), "/accounts_artifacts/ERC20_Mintable_OZ_0.8.1.json");
 
 pub const CAIRO_1_ERC20_CONTRACT_CLASS_HASH: &str =
-    "0x02ce20303e4b963bed8f11fdb309f859822ee17be99399a9acdc289443bc9d09";
+    "0x046ded64ae2dead6448e247234bab192a9c483644395b66f2155f2614e5804b0";
```

Comment:
Add a comment explaining why this is hardcoded, possibly providing the OZ link

Reply:
Added in ff6daa2769de7b9a4b1bc32a2fcac4676083ca4c

Reply:
Please don't kill me, but missing one slash to make it triple instead of double in L27.